src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('.population');
+let total = 0;
+
+for (const population of populations) {
+  const number = +population.textContent.split(',').join('');
```

Comment:
use `Number` instead `+`
```suggestion
  const number = Number(population.textContent.split(',').join(''));
```